packages/clarity-js/src/layout/dom.ts

```diff
@@ -50,15 +49,10 @@ export function extractRegions(root: ParentNode): void {
     for (let key in config.regions) {
         // We check for regions in the beginning (document) and later whenever there are new additions or modifications to DOM (mutations)
         // Since mutations may happen on leaf nodes too, e.g. textnodes, which may not support all selector APIs.
```

Comment:
this gets called before dom discovery - is it possible that between this and when mutation observer is called (there could be other elements that are added and not tracked as a part of region?)